.rubocop.yml

```diff
@@ -11,7 +11,7 @@ inherit_mode:
   merge:
     - Exclude
 
-require: rubocop-performance
+plugins: rubocop-performance
```

Comment:
The documentation details that this should be updated to use `plugins` instead of `require`: https://docs.rubocop.org/rubocop/1.75/plugin_migration_guide.html